blink/handout.tex

```diff
@@ -66,79 +59,96 @@
 
 \begin{document}
 
-\plpage
-
 \section*{Blinking LED with MicroPython on ESP32}
 
 \begin{enumerate}
 \item Set up your environment. Ask booth staff for the proper handout.
 
 \end{enumerate}
 
-\startsection{Controlling the LED.}
+\section*{Controlling the LED}
 
 \begin{enumerate}[resume]
 
-\item MicroPython allows you to read or control several pins.
-      To do that, you need to import the \texttt{Pin} class
-      with \texttt{from machine import Pin}.
+\item MicroPython allows you to control the microcontroller's \emph{pins},
+      to which other devices are connected.
+      To do that, you need to import the \texttt{Pin} class with:
+
+      \texttt{from machine import Pin}.
 
-\item A pin can serve as a binary data input or output.
-      On hardware side, imagine there either is electricity or there isn't.
-      On software side you can either read \emph{zero} or \emph{one} from the pin
-      or write that information into it.
-      To write the information you initialize a \texttt{Pin} object with
-      \texttt{Pin(number, Pin.OUT)}.
-      The integrated \emph{Light Emitting Diode} (LED) is available on pin number 2.
-      Save a variable with the pin, such as \texttt{led = Pin(2, Pin.OUT)}
+\item A pin can serve as a binary input/output.
+      On the hardware side, imagine there either is electricity or there isn't.
+      On the software side, you can either read \texttt{0} or \texttt{1} from an
+      input pin, or write \texttt{0} or \texttt{1} to an output pin.
 
-\item Set 1 to turn the light on: \texttt{led.value(1)}
+      Create an output Pin object for the integrated LED (\emph{Light Emitting Diode}),
+      which is connected to pin \#2, and save it in a variable:
 
-\item Set 0 to turn the light off: \texttt{led.value(0)}
+      \texttt{led = Pin(2, Pin.OUT)}
+
+\item Write 1 or 0 to turn the light on or off:
+
+    \texttt{
+        led.value(1) \\
+        led.value(0)
+    }
```

Comment:
![_](https://user-images.githubusercontent.com/2401856/51481369-018e9e80-1d94-11e9-9855-aa7d175fd5c1.png)

Not a blocker for now, but we should do this differently. Perhaps with minted.

Reply:
yup, leave this for later